README.md

```diff
@@ -265,13 +275,21 @@ view the [script options output][script_options] for the latest release.
           cmd: phylum-ci --lockfile requirements-prod.txt
           # Thresholds for the five risk domains may be set at the Phylum project level.
           # They can be set differently for CI environments to "fail the build."
-          # NOTE: The shortened form is used here for brevity, but the long form might be more
-          #       descriptive for future readers. For instance `--vul-threshold` instead of `-u`.
-          cmd: phylum-ci -u 60 -m 60 -e 70 -c 90 -o 80
+          cmd: >
+            phylum-ci --vul-threshold 60 --mal-threshold 60 --eng-threshold 70
+            --lic-threshold 90 --aut-threshold 80
           # Install a specific version of the Phylum CLI.
           cmd: phylum-ci --phylum-release 3.5.0 --force-install
           # Mix and match for your specific use case.
-          cmd: phylum-ci -u 60 -m 60 -e 70 -c 90 -o 80 --lockfile requirements-prod.txt --all-deps
+          cmd: >
+            phylum-ci
+            --vul-threshold 60
+            --mal-threshold 60
+            --eng-threshold 70
+            --lic-threshold 90
+            --aut-threshold 80
+            --lockfile requirements-prod.txt
+            --all-deps
```

Comment:
👍 I really like the way this looks now. Very descriptive and easy to read.

Same as above, it will have a trailing newline unless the header is changed.

```suggestion
          cmd: >-
            phylum-ci
            --vul-threshold 60
            --mal-threshold 60
            --eng-threshold 70
            --lic-threshold 90
            --aut-threshold 80
            --lockfile requirements-prod.txt
            --all-deps
```